src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -24,7 +24,9 @@ public class Program
     {
         private const string Utf8Option = "-utf8";
         private const string ForceEnglishOutputOption = "-forceEnglishOutput";
+#if DEBUG
```

Comment:
I think these are built into the product on purpose.

Reply:
`DebugOption ` variable was only used in `Debug` mode. Build failed in release mode because of `CA1801: Review unused parameters` analyzer warning.

https://github.com/NuGet/NuGet.Client/blob/144bfe8335e8d3d39fc935b1516dd0fe0cfb5871/src/NuGet.Clients/NuGet.CommandLine/Program.cs#L57-L63

Please let me know if I am missing anything here.

Reply:
nope, my bad, I missed the conditional switch.